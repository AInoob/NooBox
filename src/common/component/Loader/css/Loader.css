.Loader {
  height: 200px;
  text-align: center;
}
