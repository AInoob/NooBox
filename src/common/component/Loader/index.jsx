import React from 'react';
import { Spin } from 'antd';
import './css/Loader.css';

export default props => <Spin size="large" {...props} />;
