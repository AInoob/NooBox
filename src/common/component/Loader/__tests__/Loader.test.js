import React from 'react';
import { shallow } from 'enzyme';
import Loader from '..';

describe('Loader', () => {
  it('render correctly', () => {
    const wrapper = shallow(<Loader />);
    expect(wrapper).toMatchSnapshot();
  });
});
