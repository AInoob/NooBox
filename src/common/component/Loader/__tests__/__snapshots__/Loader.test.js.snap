// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader render correctly 1`] = `
<Spin
  size="large"
  spinning={true}
  wrapperClassName=""
/>
`;
